tests/plugins/colorbutton/manual/automaticcolor.md

```diff
@@ -0,0 +1,23 @@
+@bender-tags: bug, 4.10.1, 1084
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, sourcearea
+
```

Comment:
It would be great to use `@bender-ui: collapsed` option. When I first time looked at this test I couldn't find the source button.